kedro-telemetry/tests/test_masking.py

```diff
@@ -9,7 +9,6 @@
 from kedro import __version__ as kedro_version
 from kedro.framework.cli.cli import KedroCLI, cli
 from kedro.framework.startup import ProjectMetadata
```

Comment:
Same issue with discovering that `kedr_telemetry` is the 'active' module.

The linter should get directions which folder is the `src` folder for which package

Reply:
Fixed.